tools/aequatus/datatypes/aequatus_datatypes/datatypes_conf.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<datatypes>
+  <registration converters_path="lib/galaxy/datatypes/converters" display_path="display_applications">
+    <datatype extension="aequatus" type="galaxy.datatypes.text:Aequatus" mimetype="application/json" display_in_upload="true"/>
```

Comment:
We need a proper datatype because in the visualizations config file `config/plugins/visualizations/aequatus/config/aequatus.xml` the `<test type="isinstance" test_attr="datatype" result_type="datatype">` should contain a class.